packages/theme-chalk/src/drawer.scss

```diff
@@ -45,6 +45,12 @@ $directions: rtl, ltr, ttb, btt;
     font-size: 1rem;
   }
 
+  @include e(footer) {
+    padding: var(--el-drawer-padding-primary);
+    padding-top: 10px;
+    text-align: right;
```

Comment:
I would suggest that we don't use `text-align` in our stylesheets. Also the `padding-top: 10px` will have no effect.

Reply:
Though `padding-bottom` does.

Reply:
` text-align` default be left: 
![image](https://user-images.githubusercontent.com/21104054/149700365-8b250c25-83f7-41ae-9238-017423363df1.png)

Reply:
In fact, I directly refer to the dialog's style sheet
![image](https://user-images.githubusercontent.com/21104054/149700772-02cb3061-6141-469e-8508-bbffe3cf83fb.png)

Reply:
` padding-top: 10px`  have effect.
![image](https://user-images.githubusercontent.com/21104054/149701375-0da1631a-dfeb-46de-bdf3-e6358c3d835e.png)
![image](https://user-images.githubusercontent.com/21104054/149701610-dfb9039b-c343-4e6f-9ded-d3094cc683f8.png)
![image](https://user-images.githubusercontent.com/21104054/149701403-17bafb00-7492-4cf5-95d7-530ff1651bc0.png)

remove it :
![image](https://user-images.githubusercontent.com/21104054/149701431-b5cbc23e-7771-4578-b403-f3098fdcc4c3.png)
![image](https://user-images.githubusercontent.com/21104054/149701553-6ba61530-23bc-47fc-a399-b49e24264fae.png)

![image](https://user-images.githubusercontent.com/21104054/149701504-f3970a2f-5b22-4bfb-ac81-f0c502ad06a6.png)

Reply:
Yes, the concern is understood, humm, however, let's keep this way for now, see if there is any feedbacks from our users.